### PR TITLE
Random Docs Updates

### PR DESCRIPTION
**Pull request by kendallstrautman**
## what this does

- removes duplicate sidebar / toolbar config options from cms overview
- makes fields optional in `useForm` config
- rephrases the ui introduction to better capture the other ui components involved in creating an editing interface (i.e. not just sidebar / toolbar)

### CODE REVIEW

File: content/docs/cms.md [hunk 1]
```diff
@@ -93,13 +95,13 @@ export default function SomeComponent() {
 
 ## Disabling / Enabling the CMS
 
-The CMS can be enabled or disabled via methods or configuration on the CMS object.
+The CMS can be enabled or disabled via methods or configuration on the CMS object. Disabling the CMS prevents editing UI from rendering and forms from registering.
```
Comment (dwalkr) [inline content/docs/cms.md:98]:
```suggestion
The CMS can be enabled or disabled via methods or configuration on the CMS object. Disabling the CMS prevents the editing UI from rendering and forms from registering.
```

**Comment by dwalkr** [comment]
I think making fields optional is a good idea, but it doesn't look like they currently are and I don't see a PR open to make that change. I think we should hold off on changing the docs until that is changed + released

File: content/docs/ui.md [hunk 2]
```diff
@@ -9,9 +9,15 @@ consumes:
     details: Shows toolbar interface
 ---
 
-The `tinacms` package provides two possible user interfaces: the sidebar and the toolbar.
+There are two main pieces of UI that can construct or supplement the content editing interface: the sidebar and the toolbar.
```
Comment (dwalkr) [inline content/docs/ui.md:12]:
```suggestion
There are two main UI components that support the content editing interface: the sidebar and the toolbar.
```
just a suggestion to make this sentence a bit simpler

File: content/docs/plugins/forms.md [hunk 3]
```diff
@@ -102,7 +102,7 @@ In addition to Final Form's options, Tina's form hooks accept the following addi
 interface FormOptions<S> {
   id: any
   label: string
-  fields: Field[]
+  fields?: Field[]
```
Comment (kendallstrautman) [inline content/docs/plugins/forms.md:105]:
```suggestion
  fields: Field[]
```

File: content/docs/plugins/forms.md [hunk 4]
```diff
@@ -116,11 +116,11 @@ interface FormOptions<S> {
...
-| key | description |
-| --- | --- |
-| `id` | A unique identifier for the form. This should be derived from the content to distinguish it from other instances of the form. |
-| `label` | A label for the form that will appear in the sidebar. |
-| `fields` | An array of fields that will define the shape of the form and how content is edited. |
+| key                 | description                                                                                                                                   |
+| ------------------- | --------------------------------------------------------------------------------------------------------------------------------------------- |
+| `id`                | A unique identifier for the form. This should be derived from the content to distinguish it from other instances of the form.                 |
+| `label`             | A label for the form that will appear in the sidebar.                                                                                         |
+| `fields`            | _Optional:_ An array of fields that will define the shape of the form and how content is edited.                                              |
```
Comment (kendallstrautman) [inline content/docs/plugins/forms.md:123]:
```suggestion
| `fields`            | An array of fields that will define the shape of the form and how content is edited.                                              |
```

**Pull request merged**